src/Compute/Compute/ChangeLog.md

```diff
@@ -22,6 +22,7 @@
 ## Upcoming Release
 
 ## Version 4.32.0
+* Updated `Get-AzHost` cmdlet logic to return Host for `-ResourceId` parameterset.
```

Comment:
Add this to the Upcoming Release section.

Reply:
I added there itself, but it was showing Conflict. So in order to resolve it I moved it here. Nvm will add it back to where it was.

Reply:
Done